lib/aca_entities/medicaid/ios/contracts/ssp_asset__c_contract.rb

```diff
@@ -0,0 +1,203 @@
+            optional(:ProviderName__C).maybe(:string)
+            optional(:IsExistingData__c).maybe(:bool)
+            optional(:IsDeleted__c).maybe(:bool)
+            optional(:RecordTypeId).maybe(:string)
+            optional(:Id).maybe(:string)
+            optional(:AccountBalance__c).maybe(:float)
+            optional(:ActivityStartDate__c).maybe(:date)
+            optional(:ActivityType__c).maybe(:string)
+            optional(:ReceiveInHouseAssistance__c).maybe(:string)
+            optional(:RecordType).maybe(:hash)
+            optional(:AddressLine1__c).maybe(:string)
+            optional(:AddressLine2__c).maybe(:string)
+            optional(:BooksAmount__c).maybe(:float)
+            optional(:BusinessTitle__c).maybe(:string)
+            optional(:BusinessTypeCode__c).maybe(:string)
+            optional(:CashSurrenderValue__c).maybe(:float)
+            optional(:ChildName__c).maybe(:string)
+            optional(:City__c).maybe(:string)
+            optional(:CountyCode__c).maybe(:string)
+            optional(:DateOfissuance__c).maybe(:date)
+            optional(:DateOwned__c).maybe(:date)
+            optional(:DCAddressId__c).maybe(:integer)
+            optional(:DCEmploymentDetailId__c).maybe(:integer)
+            optional(:DCExpenseDetailId__c).maybe(:integer)
+            optional(:DCId__c).maybe(:integer)
+            optional(:DCResourceValueDetailId__c).maybe(:integer)
+            optional(:DCResourceMiscellaneousDetailId__c).maybe(:integer)
+            optional(:DCResourceOwnershipDetailId__c).maybe(:integer)
+            optional(:DCSelfEmploymentIncomeDetailId__c).maybe(:integer)
+            optional(:DCUnearnedIncomeDetailId__c).maybe(:integer)
+            optional(:DependentCareProvider__c).maybe(:string)
+            optional(:DependentCareProviderId).maybe(:integer)
+            optional(:DependentIndividual__c).maybe(:string)
+            optional(:EIN__c).maybe(:integer)
+            optional(:EmployerName__c).maybe(:string)
+            optional(:EmploymentEndReason__c).maybe(:string)
+            optional(:EndDate__c).maybe(:date)
+            optional(:ExpenseAmount__c).maybe(:float)
+            optional(:ExpenseFrequencyCode__c).maybe(:string)
+            optional(:ExpenseSubType__c).maybe(:string)
+            optional(:ExpenseTypeCode__c).maybe(:string)
+            optional(:FaceValueAmount__c).maybe(:integer)
+            optional(:FeesAmount__c).maybe(:integer)
+            optional(:FuneralFundCode__c).maybe(:string)
+            optional(:FuneralLocation__c).maybe(:string)
+            optional(:HasItemizedStmtOfGoodsSignByClientToggle__c).maybe(:string)
+            optional(:HasPolicyAssignedToFuneralHomeToggle__c).maybe(:string)
+            optional(:IncomePayDetailHoursPerWeek__c).maybe(:integer)
+            optional(:IncomePayFrequency__c).maybe(:string)
+            optional(:IncomeTypeCode__c).maybe(:string)
+            optional(:IncomeSubtypeCode__c).maybe(:string)
```

Comment:
this is marked as required on the database, shall we set as required here ?

Reply:
Similar to the expenses, but going the other way. I'll add in some conditional validation